AdvanceSpecFlowProject/SpecFlowProject/Steps/HomePageSteps.cs

```diff
@@ -0,0 +1,106 @@
+﻿using SpecFlowProject.Model;
+using SpecFlowProject.Pages;
+using NUnit.Framework;
+using SpecFlowProject.Pages.Components.ProfileOverview;
+using SpecFlowProject.Pages.Components.ShareSkillOverview;
+using SpecFlowProject.Pages.Components.ProfileMenuTabNavigationOverview;
+
+namespace SpecFlowProject.Steps
+{
+    public class HomePageSteps
+    {
+        HomePage HomePageComponentObj;
+        ProfileOverviewComponents ProfileOverviewComponentsObj;
```

Comment:
you could just have one step package, for NUnit you did not have a steps but for specflow you do! you could just move all this step logic into step definition

Reply:
convert Steps package into Services package and rename HomePageSteps to HomePageService

Reply:
Issue resolved. Please review.